package.json

```diff
@@ -1,34 +1,34 @@
 {
-  "license": "BSD-2-Clause", 
-  "name": "@mapbox/geo-viewport", 
+  "license": "BSD-2-Clause",
+  "name": "@mapbox/geo-viewport",
   "repository": {
-    "url": "git@github.com:mapbox/geo-viewport.git", 
+    "url": "git@github.com:mapbox/geo-viewport.git",
     "type": "git"
-  }, 
-  "author": "Tom MacWright", 
+  },
+  "author": "Tom MacWright",
   "bugs": {
     "url": "https://github.com/mapbox/geo-viewport/issues"
-  }, 
-  "version": "0.2.2", 
+  },
+  "version": "0.2.2",
   "dependencies": {
-    "@mapbox/sphericalmercator": "~1.0.2"
-  }, 
+    "@mapbox/sphericalmercator": "~1.1.0"
```

Comment:
This is the meat of the PR - updating a dependency. Easy peasy!

Unfortunately, the diff is a bit noisy for the `package.json`. I'm developing on npm 6, and it insisted on removing the trailing whitespace from every line in the file. Nonetheless, I left it in, as it isn't a difficult change to follow.